evm_arithmetization/src/generation/state.rs

```diff
@@ -432,7 +483,7 @@ impl<F: Field> State<F> for GenerationState<F> {
     fn incr_interpreter_clock(&mut self) {}
```

Comment:
I think I'd rather have the trait define the blanket no-op impl, so that we don't have to keep a reference to something that's interpreter specific here.

Reply:
Actually, in the end, I think it's better to increment the interpreter clock in `push_cpu`, since this is when the generation "clock" is increased. This would get rid of that method altogether, and would insure a better unification (we noticed that the clock of the interpreter was not increased when calling `generate_exception` for example.